toolkit/scripts/toolkit.mk

```diff
@@ -101,3 +105,21 @@ $(rpms_snapshot_per_specs): $(go-rpmssnapshot) $(chroot_worker) $(local_specs) $
 
 print-build-summary:
 	sed -E -n 's:^.+level=info msg="Built \(([^\)]+)\) -> \[(.+)\].+$:\1\t\2:gp' $(LOGS_DIR)/pkggen/rpmbuilding/* | tee $(LOGS_DIR)/pkggen/build-summary.csv
+
+# This will run the spec arch checker and generate a list of valid spec names suitable for passing to PACKAGE_BUILD_LIST et. al.
+.PHONY: prepare-spec-arch-checker
```

Comment:
I think `prepare-spec-arch-checker` is not a target defined anywhere in the makefiles.

Reply:
Oops, missed the rename on that.